modules/ocf_www/manifests/site/ocfweb_redirects.pp

```diff
@@ -3,7 +3,7 @@
 class ocf_www::site::ocfweb_redirects {
   # accounts
   $accounts_canonical_url = $::hostname ? {
-    /^dev-/ => 'https://dev-accounts.ocf.berkeley.edu/',
+    $::dev_config => 'https://dev-accounts.ocf.berkeley.edu/',
```

Comment:
[This arcane syntax](https://puppet.com/docs/puppet/4.9/lang_conditional.html#selectors) does not mean what you think it means. All these matches against the host name should be replaced with simple if-else statements.